backend/danswer/connectors/sharepoint/connector.py

```diff
@@ -1,9 +1,11 @@
 import io
 import os
+from dataclasses import dataclass
+from dataclasses import field
 from datetime import datetime
 from datetime import timezone
-from typing import Any, Optional
```

Comment:
Probably we don't need this optional, typing is built in now so we can just do "| None"